Verifying sv/ldv-linux-3.16-rc1/43_2a_bitvector_linux-3.16-rc1.tar.xz-43_2a-drivers--usb--host--max3421-hcd.ko-entry_point_false-unreach-call.cil.out.c
main using GenerateFunctionCallPreprocessor ...
GenerateFunctionCallPreprocessor parents for __VERIFIER_error:
__VERIFIER_error
  ldv_error
    ldv_check_alloc_nonatomic
    <end>
    ldv_check_alloc_flags_and_return_some_page
    <end>
    ldv_check_alloc_flags
      kmalloc
        max3421_probe
          main
          <end>
        max3421_probe
          main
          <end>
      kzalloc
        max3421_urb_enqueue
          main
          <end>
          main
          <end>
      ldv_kmem_cache_alloc_16
      <end>
    ldv__builtin_trap
    <end>
